web/js/app.js

```diff
@@ -0,0 +1,9 @@
+require('jquery');
+require('../../node_modules/material-design-lite/dist/material.min.js');
+
+require('./main.js');
+
+require('../../node_modules/material-design-lite/dist/material.min.css');
```

Comment:
These shouldn't be relative to this, but instead something like: `material-design-lite/dist/material.min.css`

Reply:
Updated this